scripts/update-minor.sh

```diff
@@ -0,0 +1,15 @@
+LAST_TAG="$$TRAVIS_TAG"
+
+if [ $LAST_TAG != "" ]; then
+
+MAYOR=$(echo $LAST_TAG | cut -d'.' -f1 | tr -d '[a-z][A-Z]')
+MINOR=$(echo $LAST_TAG | cut -d'.' -f2)
+PATCH=$(echo $LAST_TAG | cut -d'.' -f3)
+
+echo "Current tag: $LAST_TAG"
+
+REPLACEMENT="VERSION $MAYOR.$MINOR.$PATCH"
+echo "Updating CMAKE version: VERSION 1.0.0 -> $REPLACEMENT"
+sed -i.bak "s/VERSION 1.0.0/$REPLACEMENT/g" CMakeLists.txt
```

Comment:
we can't use a placeholder, cmake fails, we have to use some fake version like 1.0.0.

Also, the commit as version is not a valid semantic version and it fails too